components/clubs-page.jsx

```diff
@@ -208,7 +166,29 @@ var ClubsPage = React.createClass({
           <Quote/>
         </section>
         <section>
-          <IconLinks/>
+          <IconLinks links={[
+            {
+              linkTo: "fixme",
+              imgSrc: "/img/icon-curriculum.svg",
+              imgAlt: "icon curriculum",
+              head: "Curriculum",
+              subhead: "Modular Web Literacy curriculum"
+            },
+            {
+              linkTo: "fixme",
+              imgSrc: "/img/icon-connect.svg",
+              imgAlt: "icon connect",
+              head: "Connect",
+              subhead: "Connect with other Club Leaders"
+            },
+            {
+              linkTo: "fixme",
+              imgSrc: "/img/icon-tips.svg",
+              imgAlt: "icon tips",
+              head: "Helpful Tips",
+              subhead: "Tips for running your Club"
+            }
+          ]}/>
```

Comment:
I don't think that it would be a hard dependency, based on your code, i.e. the code would still run and compile if `IconLink`s were used outside of `IconLinks`--they just might not style particularly well?

Then again, I may not have read your code closely enough yet. I was thinking `IconLinks` would essentially just be sugar for something like `<div className="col-blah myFunkyClass">{this.props.children}</div>`.